Description: Extracted from 1.3rc1-8.5's .diff.gz by diff2patches

--- bb-1.3rc1.orig/bb.c
+++ bb-1.3rc1/bb.c
@@ -24,6 +24,18 @@
 #include <string.h>
 #include <stdlib.h>
 #include <ctype.h>
+
+#if TIME_WITH_SYS_TIME
+# include <sys/time.h>
+# include <time.h>
+#else
+# if HAVE_SYS_TIME_H
+#  include <sys/time.h>
+# else
+#  include <time.h>
+# endif
+#endif
+
 #include <aalib.h>
 #include "bb.h"
 
